util/s3/s3Helper.go

```diff
@@ -34,6 +33,9 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3/s3manager"
 )
 
+const algorandReleasesBucketName = "algorand-releases"
```

Comment:
Same comments as above.  Also, we should take this opportunity to modify updater to use the shared version of the s3helper code - we don't need all the duplicate code.  I didn't have time/energy when I added this to modify updater to use it.

Reply:
OK, consolidated the 2 files into 1 within util/s3.